go/vt/vtgate/evalengine/integration/comparison_test.go

```diff
@@ -572,3 +572,14 @@ func TestCharsetConversionOperators(t *testing.T) {
 		}
 	}
 }
+
+func TestCeil(t *testing.T) {
+	var conn = mysqlconn(t)
+	defer conn.Close()
+	var input = []string{
+		"0", "1", "1.123", "-1.23", "NULL",
```

Comment:
You're not testing this with any floating point numbers. Could you please add some here? There are also other interesting test cases, such as strings, larger decimals, decimals closer to the `.5` half in rounding, larger negative decimals, numbers that would upcast their type when ceiled, etc. Please be imaginative and try to add as many test cases as possible.